distributionviewer/core/static/css/chart-detail.styl

```diff
@@ -14,4 +14,15 @@
     text-align: center;
     font-weight: 100;
   }
+
+  .options {
+    & > * {
+      float: left;
```

Comment:
I'd vote for flexboxing this since we're doing flexbox elsewhere and are not too worried about old browser support. It will also eliminate the need for the `clear` later.